Mathlib/Analysis/Complex/Conformal.lean

```diff
@@ -266,6 +266,15 @@ theorem complexOfReal_hasDerivWithinAt (h₁ : DifferentiableWithinAt ℝ f s x)
   rw [hasDerivWithinAt_iff_hasFDerivWithinAt, smulRight_one_one]
   exact (h₁.hasFDerivWithinAt).complexOfReal_hasFDerivWithinAt h₂
 
+/--
+In cases where the **Cauchy-Riemann Equation** guarantees complex differentiability at `x`, the
+complex derivative equals `ContinuousLinearMap.complexOfReal` of the real derivative.
```

Comment:
the docstring does not really correspond to the statement.

Same thing for the other lemma below.